### PR TITLE
Modify MapFragment

### PR DESCRIPTION
**Pull request by 1coin178**
## Issue
NONE

## Overview (Required)
- Fix 5 points
 1. Fixed a place you think that the typo. (lng followed the Google Map's LatLng class)
 2. TextViews is Selectable.
    - Because I want to copy texts!!
 3. add LF for improving visibility
    - ref: [アクセス | DroidKaigi 2018 | Zeplin Scene](https://goo.gl/An9yUa)
 4. fix NagivationDrawer title 
    - ref: [drawer | DroidKaigi 2018 | Zeplin Scene](https://goo.gl/dXz1Yc)
 5. format fragment_map.xml

## Screenshot
NavigationDrawer(JP) | Access Map(JP)
:--: | :--:
 | 

NavigationDrawer(US) | Access Map(US)
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/map/MapFragment.kt [hunk 1]
```diff
@@ -128,6 +128,6 @@ class MapFragment : Fragment(), Injectable, OnMapReadyCallback {
         fun newInstance(): MapFragment = MapFragment()
 
         private const val placeLat = 35.6957954
-        private const val placeLang = 139.69038920000003
+        private const val placeLng = 139.69038920000003
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/map/MapFragment.kt]: Also, if you do not mind, can you capitalize it?
https://android.github.io/kotlin-guides/style.html#constant-names

File: app/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -54,11 +54,11 @@
     <string name="feed_content_description_not_found">お知らせが無い画像</string>
 
     <!-- Map -->
-    <string name="map_title_activity">地図</string>
+    <string name="map_title_activity">アクセス</string>
```
Comment (takahirom) [inline app/src/main/res/values-ja/strings.xml]: Nice catch 👍

**Comment by takahirom** [comment]
Thanks 👍

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/map/MapFragment.kt (hunk 1 above)
Comment (1coin178) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/map/MapFragment.kt]: Oh, I understood. I'll give it a try!

**Comment by takahirom** [comment]
LGTM 👍 Thank you for cleaning code ⚡️

**Pull request merged**